src/psyclone/tests/core/single_variable_access_info_test.py

```diff
@@ -138,9 +139,12 @@ def test_variable_access_info():
     assert vai.is_read_only()
     assert vai.all_read_accesses == [vai[0]]
     assert vai.all_write_accesses == []
+    assert not vai.is_written()
+    assert not vai.is_written_first()
     vai.change_read_to_write()
     assert not vai.is_read()
     assert vai.is_written()
+    assert vai.is_written_first()
```

Comment:
Can we test when a variable is written but that access is not the first one?

Reply:
That is already done. I've added an earlier test and a comment or two to make this clear.

Reply:
Great, thanks.